src/trials.jl

```diff
@@ -338,115 +346,116 @@ Base.show(io::IO, t::TrialEstimate) = _show(io, t)
 Base.show(io::IO, t::TrialRatio) = _show(io, t)
 Base.show(io::IO, t::TrialJudgement) = _show(io, t)
 
+_percentile() = 99  # to tweak this live, TODO remove
+
 function Base.show(io::IO, ::MIME"text/plain", t::Trial)
 
     pad = get(io, :pad, "")
-    print(io, "BenchmarkTools.Trial: ", length(t), " sample", if length(t) > 1 "s" else "" end,
-          " with ", t.params.evals, " evaluation", if t.params.evals > 1 "s" else "" end ,".\n")
+    padcolor = :light_black
+
+    showpercentile = _percentile()
 
     perm = sortperm(t.times)
     times = t.times[perm]
     gctimes = t.gctimes[perm]
 
-    if length(t) > 1
-        med = median(t)
-        avg = mean(t)
-        std = Statistics.std(t)
-        min = minimum(t)
-        max = maximum(t)
-
-        medtime, medgc = prettytime(time(med)), prettypercent(gcratio(med))
-        avgtime, avggc = prettytime(time(avg)), prettypercent(gcratio(avg))
-        stdtime, stdgc = prettytime(time(std)), prettypercent(Statistics.std(gctimes ./ times))
-        mintime, mingc = prettytime(time(min)), prettypercent(gcratio(min))
-        maxtime, maxgc = prettytime(time(max)), prettypercent(gcratio(max))
-
-        memorystr = string(prettymemory(memory(min)))
-        allocsstr = string(allocs(min))
+    if length(t) == 0
+        print(io, "BenchmarkTools.Trial: 0 samples")
+        return
     elseif length(t) == 1
-        print(io, pad, " Single result which took ")
-        printstyled(io, prettytime(times[1]); color=:blue)
-        print(io, " (", prettypercent(gctimes[1]/times[1]), " GC) ")
-        print(io, "to evaluate,\n")
-        print(io, pad, " with a memory estimate of ")
-        printstyled(io, prettymemory(t.memory[1]); color=:yellow)
-        print(io, ", over ")
-        printstyled(io, t.allocs[1]; color=:yellow)
-        print(io, " allocations.")
+        println(io, "BenchmarkTools.Trial:")
+        # Time
+        print(io, pad, "│  Only 1 sample: ")
+        printstyled(io, prettytime(times[1]); color=:green)
+
+        # Memory
+        println(io)
+        print(io, pad, "│  ", prettycount(t.allocs[1]), " allocation", t.allocs[1]==1 ? "" : "s")
+        if t.allocs[1] > 0
+            print(io, ", ", prettymemory(t.memory[1]))
+        end
+
+        # GC time
+        if t.gctimes[1] > 0
+            println(io)
+            print(io, pad, "│  GC time: ", prettytime(t.gctimes[1]))
+            printstyled(io, " (", prettypercent(t.gctimes[1] / t.times[1]),")"; color=:green)
+        end
         return
+    end # done with trivial cases.
+
+    med = median(t)
+    avg = mean(t)
+    min = minimum(t)
+    max = maximum(t)
+    q99 = quantile(t, showpercentile/100)
```

Comment:
If someone changes the value of `showpercentile` then the name of the variable will be a lie. How about `qhigh` or something?

Reply:
I mostly wanted to mess around myself (and had temporarily broken Revise...). My vote is to hard-wire 99% as this seems reasonable. And was the existing right limit of the histogram.